model/clause_checker.py

```
@@ -1,5 +1,731 @@
+            logger.info(f"🔍 {'전세' if is_jeonse else '월세'} 계약 법령 검색: {search_query}")
+            
+            return search_law(search_query, k=10)
+            
+        except Exception as e:
+            logger.error(f"❌ 전체 계약서 법령 검색 실패: {e}")
+            return []
+    
+    def _parse_full_contract_analysis_improved(self, llm_result: str) -> List[LegalViolation]:
+        """개선된 전체 계약서 LLM 분석 결과 파싱"""
+        try:
+            logger.info(f"🔍 전체 계약서 분석 결과 파싱 시작")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)